test/multifile/typealias/two-modules/library.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: true
```

Comment:
The thing is a lot of the SILGen issues came up with both cross-file and cross-module things.

What about splitting up test/multifile then?